recipe/meta.yaml

```diff
@@ -1,43 +1,44 @@
-{% set version = "1.0.0" %}
+{% set name = "nose-timer" %}
+{% set version = "1.0.1" %}
+
 
 package:
-  name: nose-timer
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: nose-timer-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/n/nose-timer/nose-timer-{{ version }}.tar.gz
-  sha256: 4c383d67ae6affd67173217e23689308c1a2bb178bfaf8b05b06c4cf16971c26
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nose-timer-{{ version }}.tar.gz
+  sha256: 8f70d103b7ffd9122a589de0df9d037a7d967519bf6de122621d2186609b9e3a
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
   host:
     - pip
-    - python
-    - setuptools
-
+    - python >=3.5
   run:
-    - python
     - nose
-    - termcolor
```

Comment:
Can we please keep `termcolor`? It is [optional]( https://github.com/mahmoudimus/nose-timer/blob/aabc31c2687c5fe0031dab6a781e9b3070472c29/nosetimer/plugin.py#L16-L19 ), but useful for seeing what is faster or slower.

Also may be worth [adding `colorama`]( https://github.com/mahmoudimus/nose-timer/blob/aabc31c2687c5fe0031dab6a781e9b3070472c29/nosetimer/plugin.py#L21-L29 ) as well

Reply:
Ahh yeah sorry one sec.